### ISSUE TITLE
Memory allocation failure

### ISSUE THREAD

**Issue by 5hadowblad3** [usi-verification-and-security/opensmt#112]
Hi, there.

I encounter a memory allocation failure with the attached input.
To reproduce it, just run
~~~~
./opensmt input.smt2
~~~~

This is the error information.
~~~~
opensmt: malloc.c:2868: mremap_chunk: Assertion `((size + offset) & (GLRO (dl_pagesize) - 1)) == 0' failed.
~~~~

[opensmt-malloc1.smt2.zip](https://github.com/usi-verification-and-security/opensmt/files/4689213/opensmt-malloc1.smt2.zip)

**Comment by rainoftime** [comment]
In debug mode, it trigged an assertion violation
~~~~
opensmt: /opensmt/src/logics/Logic.cc:2040: void Logic::conjoinItes(PTRef, PTRef&): Assertion `id < size' failed.
Aborted
~~~~

**Comment by blishko** [comment]
Thanks for the report, we have a fix ready and will integrate it soon.

**Comment by blishko** [comment]
Fixed in commit 5b27a03, #114
